src/main/webapp/src/components/autocompleteTextField.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { withStyles } from "@material-ui/styles";
+import TextField from '@material-ui/core/TextField';
+import Autocomplete from '@material-ui/lab/Autocomplete';
+import Typography from '@material-ui/core/Typography';
+import throttle from 'lodash/throttle';
+import axios from "axios";
+
+/**
+ * Change TextField outline border default colors.
+ */
+const StyledTextField = withStyles({
+  root: {
+    "& label.Mui-focused": {
+      color: "black",
+    },
+    "& .MuiInput-underline:after": {
+      borderBottomColor: "#733F94",
+    },
+    "& .MuiOutlinedInput-root": {
+      "&.Mui-focused fieldset": {
+        borderColor: "#733F94",
+      },
+    },
+  },
+})(TextField);
```

Comment:
Up to you, but I think it'd be more organized to keep the styles in a css file

Reply:
Will be done in #30